apps/test/solutions/applab/ec_screens.js

```diff
@@ -520,5 +520,53 @@ module.exports = {
         testResult: TestResults.FREE_PLAY
       },
     },
+    {
+      description: "load a level with two screens",
+      editCode: true,
+      xml: '',
+      levelHtml: '<div xmlns="http://www.w3.org/1999/xhtml" id="designModeViz" style="width: 320px; height: 450px; display: block;">' +
+                   '<div class="screen" tabindex="1" id="screen1" style="display: block; height: 450px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0;">' +
+                     '<button id="button1" style="padding: 0px; margin: 0px; height: 30px; width: 80px; font-size: 14px; color: rgb(255, 255, 255); position: absolute; left: 160px; top: 70px; background-color: rgb(26, 188, 156);">Button</button>' +
+                   '</div>' +
+                   '<div class="screen" tabindex="1" id="screen2" style="display: none; height: 450px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0;">' +
+                     '<button id="button2" style="padding: 0px; margin: 0px; height: 30px; width: 80px; font-size: 14px; color: rgb(255, 255, 255); position: absolute; left: 185px; top: 65px; background-color: rgb(26, 188, 156);">Button</button>' +
+                   '</div>' +
+                 '</div>',
```

Comment:
my approach with these is usually to put an empty string on line 1, and then just have all the subsequent lines indented just once.